tuna/miopen/subcmd/import_configs.py

```diff
@@ -106,7 +55,13 @@ def create_query(tag, mark_recurrent, config_id):
   return query_dict
 
 
-def tag_config_v2(driver, counts, dbt, args, new_cf=None):
+def tag_config_v2(
+    driver: Union[DriverConvolution, DriverBatchNorm],
```

Comment:
Do type-hints support polymorphism, like declaring with DriverBase?

Reply:
seems like it does, I have updated